geo-data-server/src/main/resources/db/changelog/changesets/changelog_2022-04-04T12:24:52Z.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-3.10.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.10.xsd">
+    <changeSet author="lecuyerfra (generated)" id="1649075101209-1">
+        <addColumn tableName="line_entity">
+            <column name="coordinates" type="varchar"/>
```

Comment:
maybe add an initial value [] ? otherwise we you forget to reimport data you get 

```
java.lang.IllegalArgumentException: argument "content" is null
  at com.fasterxml.jackson.databind.ObjectMapper._assertNotNull(ObjectMapper.java:4429) ~[jackson-databind-2.10.5.1.jar:2.10.5.1]
  at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3181) ~[jackson-databind-2.10.5.1.jar:2.10.5.1]
```